components/dialog/README.md

```diff
@@ -45,24 +45,20 @@ document.querySelector('#open').addEventListener('click', () => {
   });
 });
 ```
-Make sure that in the 'd2l-dialog-close' event handler that you sync any bound properties if you decided to use them:
+
+*Important:* The user may close the dialog in a few different ways: clicking the dialog workflow buttons (marked up with `data-dialog-action`); clicking the `[x]` button in the top-right corner; or pressing the `escape` key. Therefore, if your component tracks the `opened` state of the dialog with its own property, it's important to keep it in sync by listening for the `d2l-dialog-close` event.
```

Comment:
Simplified this by combining with the note that followed.  Hopefully it's clearer.